com.hoptodesk.HopToDesk.yml

```diff
@@ -0,0 +1,52 @@
+id: com.hoptodesk.HopToDesk
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: hoptodesk
+icon: share/icons/hicolor/scalable/apps/hoptodesk.svg
```

Comment:
```suggestion
```

Reply:
This is not a real Flatpak manifest syntax...

Reply:
Also which documentation said to do use `icon` like that? I'd like to make sure it is fixed because it is incorrect.